website/project/views/contributor.py

```diff
@@ -541,8 +541,8 @@ def notify_added_contributor(node, contributor, auth=None, throttle=None, email_
 
     throttle = throttle or settings.CONTRIBUTOR_ADDED_EMAIL_THROTTLE
     # Email users for projects, or for components where they are not contributors on the parent node.
-    if contributor.is_registered and (isinstance(node, Preprint) or
-            (not node.parent_node or (node.parent_node and not node.parent_node.is_contributor(contributor)))):
+    if contributor.is_registered and ((isinstance(node, Preprint) or (isinstance(node, DraftRegistration)) or
```

Comment:
You can shorten this to `isinstance(node, (Preprint, DraftRegistration))` for example `isinstance('hi!', (int, str)) == True`